### PR TITLE
Update INSTALL.md and LOCALIZE.md

### PR DESCRIPTION
**Pull request by Shailin**
|PR checklist|Status|
|-----|:------:|
|Is issue created in GitHub?|[Yes](https://github.com/maqsoftware/Pehla-Class/issues/215)|
|Is code analysis task executed and issues resolved in the Java/Kotlin code?|NA|
|Is code analysis task executed and issues resolved in the CPP/native code?|NA|
|Is the version code set in the AndroidManifest.xml or build.gradle file to 1 more than the version which has already been published?|NA|
|Is code commenting done properly for the changes made?|NA|
|Is correct variable naming convention followed?|NA|
|Is auto formatting shortcut Ctrl+Alt+Shift+L executed in all files?|NA|
|Are the message strings used in the application taken from strings.xml file?|NA|
|Are the permissions which are not required removed from AndroidManifest.java file?|NA|
|Is the _uses-feature_ tag used for only must have feature in AndroidManifest.java file?|NA|
|Are the generated files correctly configured in .gitignore?|NA|
|Are the release signing configurations removed from the gradle files?|NA|
|Is commented code removed from all files?|NA|
|Are updated images compressed (< 100 KB)?|NA|
|Ensured there are no typos in any string. For example: Variable/method names, hardcoded strings, commit messages, PR titles, issues, code comments?|Yes|

### CODE REVIEW

File: INSTALL-STANDALONE.md [hunk 1]
```diff
@@ -24,59 +20,36 @@ The installation steps below will allow you to set-up the application as a stand
...
-2. Configure the _ROOT_FOLDER_ environment variable and _ANDROID_HOME_ environment variable based on the location of the Android SDK:  
-   
-   Follow these steps for macOS (Mavericks or later) or Linux-based system:
-	
-		export ROOT_FOLDER=onecourse
-		
-		export ANDROID_HOME=[PATH/TO/ANDROID_SDK]
-		
-   Follow these steps for a Windows 7 or later system:
+3. To change the app locale, just select the desired language from the build variant.
```
Comment (akashdabhi03) [inline INSTALL-STANDALONE.md]: In order to build the app for a specific language, select the desired language from the build variant.

File: INSTALL-STANDALONE.md [hunk 2]
```diff
@@ -24,59 +20,36 @@ The installation steps below will allow you to set-up the application as a stand
...
-		Set-Variable -Name "ROOT_FOLDER" -Value "onecourse"
-		
-		Set-Variable -Name "ANDROID_HOME" -Value "PATH/TO/ANDROID_SDK"
-		
-3. Clone the **onecourse source** repository:
-		
-		git clone https://github.com/XPRIZE/GLEXP-Team-onebillion.git $ROOT_FOLDER/
-		
-4. In the _ROOT_FOLDER_ build _gradle_ tasks:
+4. Add google-services.json file in language specific [app](https://github.com/maqsoftware/Pehla-Class/tree/master/app/src) folder. It is there in Firebase console after the app project has been created.
```
Comment (akashdabhi03) [inline INSTALL-STANDALONE.md]: The _google-service.json_ file is available in the Firebase console after a new Android app has been added to a Firebase project.

File: INSTALL-STANDALONE.md [hunk 3]
```diff
@@ -24,59 +20,36 @@ The installation steps below will allow you to set-up the application as a stand
...
-## 3. Installing onecourse
+## 3. Installing Pehla Class
 
 1. Install the application via adb:
 
-	onecourse English:
+	Pehla Class English:
 
-		adb install app/build/outputs/apk/enGB_community_/debug/app-enGB_community_-debug.apk
+		adb install app/english/release/app-english-release.apk
```
Comment (akashdabhi03) [inline INSTALL-STANDALONE.md]: I think the users would select the debug version by default. Hence, we should refer to the debug version folder and APK.

File: INSTALL-STANDALONE.md [hunk 4]
```diff
@@ -1,10 +1,6 @@
-# Building onecourse
+# Building Pehla Class
```
Comment (akashdabhi03) [inline INSTALL-STANDALONE.md]: We can rename this file to INSTALL.md

**Review by akashdabhi03 — CHANGES_REQUESTED** [review]

File: LOCALIZE.md [hunk 5]
```diff
@@ -137,7 +136,7 @@ The child's _learning journey_ is an ordered set of `learning units` to be worke
 
 #### Community
 
-This is defined in the file `community_LOCALE/units.xml`. An example of the first part of the onecourse Swahili _learning journey_ from `community_swunits.xml`is shown below. The First unit is an introduction to using the tablet, the second a _flashcard_ reading activity:
+This is defined in the file `community_LOCALE/units.xml`. An example of the first part of the Pehla Class Swahili _learning journey_ from `community_swunits.xml`is shown below. The First unit is an introduction to using the tablet, the second a _flashcard_ reading activity:
```
Comment (alankrutap01) [inline LOCALIZE.md]: Either keep First as in lowercase or keep second in upper case

File: LOCALIZE.md [hunk 6]
```diff
@@ -207,10 +206,10 @@ This is defined in the file `playzone_LOCALE/units.xml`. It specifies which unit
...
-onecourse by default uses two fonts, `onebillionreader-Regular.otf` and `onebillionwriter-Regular.otf`. These can be replaced by identically named alternative fonts in `app/src/main/fonts/`. Please note onecourse does not currently support right-to-left scripts.
+Pehla Class by default uses two fonts, `onebillionreader-Regular.otf` and `onebillionwriter-Regular.otf`. These can be replaced by identically named alternative fonts in `app/src/main/fonts/`. Please note Pehla Class does not currently support right-to-left scripts.
 
 ### 8. Video Subtitles
-For video clips in the onecourse _play zone_, optional subtitles can be added. These are standard `.srt` text files placed in the `assets/oc-video/local/LOCALE/` directory. Each subtitle entry within a file consists of four parts:
+For video clips in the Pehla Class _play zone_, optional subtitles can be added. These are standard `.srt` text files placed in the `assets/oc-video/local/LOCALE/` directory. Each subtitle entry within a file consists of four parts:
 
 1. A numeric counter identifying each sequential subtitle.
 2. The time that the subtitle should appear on the screen, followed by `-->` and the time it should disappear.
```
Comment (alankrutap01) [inline LOCALIZE.md]: The time at which the subtitle....

**Review by akashdabhi03 — DISMISSED** [review]

File: INSTALL.md [hunk 7]
```diff
@@ -1,81 +1,55 @@
...
+     
+2. Start Android Studio and import the project into it.
 
-## Flash filesystem images
+3. In order to build the app for a specific language, select the desired language from the build variant.
 
-1. Download the set of filesystem images for the version of onecourse you wish to install:
- - onecourse Swahili [part 1](https://github.com/XPRIZE/GLEXP-Team-onebillion/releases/download/v3.0.1/onecourse-swahili-v3.0.1.tar.gz.aa), [part 2](https://github.com/XPRIZE/GLEXP-Team-onebillion/releases/download/v3.0.1/onecourse-swahili-v3.0.1.tar.gz.ab)
- - onecourse English [part 1](https://github.com/XPRIZE/GLEXP-Team-onebillion/releases/download/v3.0.1/onecourse-english-v3.0.1.tar.gz.aa), [part 2](https://github.com/XPRIZE/GLEXP-Team-onebillion/releases/download/v3.0.1/onecourse-english-v3.0.1.tar.gz.ab)
+4. Add google-services.json file in language specific [app](https://github.com/maqsoftware/Pehla-Class/tree/master/app/src) folder. The google-service.json file is available in the Firebase console after a new Android app has been added to a Firebase project.
```
Comment (ankitk20) [inline INSTALL.md:32]: `google-services.json`. Also, the s is missing in one of its name instance.
Update the line to this "..  after registering a new app in Firebase console"

File: README.md [hunk 8]
```diff
@@ -1,39 +1,7 @@
...
-- colour-coded sections on Reading, Writing, Numeracy, Design, and Technology 
-- a summary of evidence from trials and pilot tests which were carried out in several countries, to evaluate the efficacy of onecourse 
-- information about onebillion – the non-profit organisation which produced onecourse – and the work we do
-- a set of appendices with further details about the course, our research and the software.
-
-
-## Building onecourse
-
-Please see our [detailed build instructions](BUILD.md)
+[Installation instructions for Pehla Class English](INSTALL-STANDALONE.md)
```
Comment (ankitk20) [inline README.md]: Check the reference of INSTALL-STANDALONE.md file

**Review by ankitk20 — CHANGES_REQUESTED** [review]

**Review by akashdabhi03 — APPROVED** [review]

**Review by ankitk20 — APPROVED** [review]

**Pull request merged**